vale/Grafana/Paragraphs.yml

```diff
@@ -0,0 +1,7 @@
+extends: script
+scope: raw
+level: error
+link: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element
+message: |
+  br elements must be used only for line breaks that are actually part of the content, as in poems or addresses.
```

Comment:
```suggestion
  `br` elements must be used only for line breaks that are actually part of the content, as in poems or addresses.
  
  Outside of tables, replace the use of `br` elements with semantic Markdown.
  In tables, use semantic HTML elements like `p` for paragraphs, and `ul` or `ol` for lists.
```

Reply:
> In tables, use semantic HTML elements like `p` for paragraphs, and `ul` or `ol` for lists.

Should this maybe include the `<` and `>`?

Reply:
I'm open to it, I probably switch every time I write something.

Google seems to omit the brackets: https://developers.google.com/style/semantic-tagging
MDN includes them: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p